package.sh

```diff
@@ -173,6 +173,15 @@ do_build() {
     echo "Build completed successfully."
 }
 
+# return a list of package dependencies for fpm
+get_fpm_pkg_deps() {
+    local deps=" "
+    for d in ${PKG_DEPS[@]}; do
+	deps="${deps} -d '${d}'"
+    done
+    echo -n ${deps}
```

Comment:
D'oh. I never saw that technique before for returning output from a function, interesting.

Always good to learn something new. :-)